soc/nordic/nrf53/Kconfig

```
@@ -8,6 +8,7 @@ config SOC_SERIES_NRF53X
 	select ARM
 	select SOC_COMPATIBLE_NRF53X
 	select SOC_EARLY_INIT_HOOK
+	select SOC_LATE_INIT_HOOK
```

```suggestion
	select SOC_LATE_INIT_HOOK if SOC_NRF53_RTC_PRETICK
```
would also work and remove the #ifdef from below

---

Note that it would have to be || CONFIG_SOC_NRF53_CPUNET_ENABLE.

Knowing that, do you still want it to be configurable?

---

saw that in the later commit, would probably make sense to only enable the Kconfig if one of the require features is neede

---

fixed